docs/book/src/developer/providers/contracts/control-plane.md

```diff
@@ -348,25 +348,25 @@ type FooControlPlaneStatus struct {
     // +optional
     Selector string `json:"selector,omitempty"`
     
-    // replicas is the total number of non-terminated machines targeted by this control plane
+    // replicas is the total number of machines targeted by this control plane
     // (their labels match the selector).
     // +optional
-    Replicas int32 `json:"replicas"`
+    Replicas *int32 `json:"replicas"`
```

Comment:
Looking at our KCP structs we have now (and specifically beta2). It seems like we are using omitempty 

(same below)